### DISCUSSION TITLE
How do I start the timer without having to open the window?

### DISCUSSION THREAD

**Discussion by mainrs** [nvzone/timerly#8] [Q&A] [answered]
I wanted to toggle the timer status using a keymap binding. I called `require('timerly.api').togglestatus()` but that fails due to `v` being nil. I assume that, internally, the timer relies on the buffer to exist and work properly.

**Comment by siduck** [comment, marked as answer]
you cant. this plugin's only for the UI part. there are better plugins if u just want the "time" in statusline or somewhere..

**Reply by mainrs** [reply]
Thanks!
